app/src/main/java/com/arctouch/movies/serverapi/RequestLogLevel.java

```diff
@@ -0,0 +1,41 @@
+package com.arctouch.movies.serverapi;
+
+import okhttp3.logging.HttpLoggingInterceptor;
+
+/**
+ * Created by marcelo.alves on 06/09/17.
+ */
+
+public enum RequestLogLevel {
```

Comment:
Same as my previous comment, this enum is also never used.

Reply:
This is my initial code structure for any project :) ... Thinking for next versions